roles/rpm_version/tasks/version.yml

```diff
@@ -0,0 +1,22 @@
+---
+# vim: set ft=ansible:
+#
+# This role takes a rpm and gets the version and stores it in a dictionary
+# named atomic_host
+#
+# Parameters:
+#   rv_rpm (required) - rpm installed on atomic host
+#
+- name: Fail if rv_version is not defined
+  fail:
+    msg: "rv_rpm is not defined"
+  when: rv_rpm is undefined
+
+- name: Determine version of {{ rv_rpm }}
+  command: rpm -q --queryformat '%{VERSION}' {{ rv_rpm }}
+  register: rpmq
+
+- name: Set facts
+  set_fact:
+    rv_version: "{{ rpmq.stdout }}"
+    atomic_host: "{{ atomic_host|default({}) | combine( { rv_rpm: rpmq.stdout } ) }}"
```

Comment:
This this variable is going to be globally available, it should probably be prefaced with `g_`